hack/deploy/install.sh

```diff
@@ -23,27 +116,21 @@ export PORT=8080
 export CATALOG_PATH="/etc/config/catalogs"
 export CATALOG_NAMES="kubedb"
 export STORAGE_CLASS=standard
+export UNINSTALL=0
```

Comment:
Usually I added a prefix to all ENV vars used by installer script to avoid collision with any existing env var. This also makes it easy to dump them.


https://github.com/appscode/voyager/blob/master/hack/deploy/voyager.sh#L431